components/_share.scss

```diff
@@ -0,0 +1,170 @@
+.sharing-info {
+  clear: both;
+  margin-top: 50px;
+
+  @include span-columns(8 of 12);
+  @include shift(2);
+
+  h2 {
+    margin-bottom: $base-font-size;
+  }
+}
+
+.sharing-box {
+  clear: both;
+  border: 2px solid #F0F0F1;
+  margin: auto;
+  padding: 20px;
+  background-color: #FFFFFF;
+
+  @include media($desktop) {
+    width: 465px;
+  }
+
+  .sharing-preview {
+    border: 1px solid #FCE0D7;
+    border-radius: 4px;
+
+    -webkit-animation-duration: 1s;
+    animation-duration: 1s;
+    -webkit-animation-fill-mode: both;
+    animation-fill-mode: both;
+    -webkit-animation-duration: .75s;
+    animation-duration: .75s;
+
+    &.sharing-preview-bounce {
+      -webkit-animation-name: previewBounceIn;
+      animation-name: previewBounceIn;
+    }
+
+    img#sharing-preview-image {
+      max-width: 100%;
+      display: block;
+      -webkit-border-top-left-radius: 4px;
+      -webkit-border-top-right-radius: 4px;
+      -moz-border-radius-topleft: 4px;
+      -moz-border-radius-topright: 4px;
+      border-top-left-radius: 4px;
+      border-top-right-radius: 4px;
+    }
+
+    .sharing-preview-text {
+      padding: 20px;
+      font-size: 12pt;
+      line-height: 1.5em;
+    }
+  }
+
+  .sharing-facebook-button, .sharing-try-button {
+    display: block;
+    margin-top: 20px;
+    outline: 0;
+
+    button {
+      display: block;
+      width: 100%;
+      -webkit-transition: background-color 0.25s;
+      transition: background-color 0.25s;
+      outline: 0;
+    }
+  }
+
+  .sharing-facebook-button button {
+    background-color: #435F9D;
```

Comment:
Color literals like `#435F9D` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#435F9D` should be written as `#435f9d`